README.md

```diff
@@ -1,2 +1,41 @@
-# comp110-worksheet-5
-Base repository for COMP110 worksheet 5
+comp110-worksheet-5
+===================
+
+>   by Alper Mimoglu
+
+>   Questions
+
+A.  This algorithm performs the task of finding a number in a list and when it
+    finds the number it performs a check if the correct number is true. If not,
+    it would keep looping the entire list until it finds the correct number.
```

Comment:
What do you mean by "correct number"? The answer I was looking for is that it checks the list for any duplicate elements.